plugins/events-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,182 @@
+            - statusCode
+      required:
+        - error
+        - request
+        - response
+  securitySchemes:
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+paths:
+  /bus/v1/events:
+    post:
+      operationId: PostEvent
+      description: Publish a new event
+      responses:
+        '201':
+          description: The event was published successfully
+        '204':
+          description: The event did not need to be published as all subscribers have already been notified
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      security:
+        - {}
+        - JWT: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - event
+              properties:
+                event:
+                  $ref: '#/components/schemas/Event'
+                notifiedSubscribers:
+                  type: array
+                  description: The IDs of subscriptions that have already received this event
+                  items:
+                    type: string
+            examples:
+              Publishing a simple Event:
+                value:
+                  event:
+                    topic: test-topic
+                    payload:
+                      myData: foo
+
+  /bus/v1/subscriptions/{subscriptionId}:
+    put:
+      operationId: PutSubscription
+      description: Ensures that the subscription exists with the provided configuration
+      responses:
+        '201':
+          description: The subscription exists or was created successfully
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      security:
+        - {}
+        - JWT: []
+      parameters:
+        - $ref: '#/components/parameters/subscriptionId'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - topics
+              properties:
+                topics:
+                  type: array
+                  description: The topics to subscribe to
+                  items:
+                    type: string
+            examples:
+              Subscribing to a single topic:
+                value:
+                  topics:
+                    - test-topic
+
+  /bus/v1/subscriptions/{subscriptionId}/events:
+    get:
+      operationId: GetSubscriptionEvents
+      description: Get new events for the provided subscription
+      responses:
+        '200':
+          description: New events
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - events
+                properties:
+                  events:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Event'
```

Comment:
Not really, but I don't think there's a need so far since current usage is symmetrical. We can update this after the fact though since this is effectively internal